android/tangram/src/com/mapzen/tangram/FontFileParser.java

```diff
@@ -47,6 +52,15 @@ private void processDocument(XmlPullParser parser) throws XmlPullParserException
 
                         String key = name + "_" + weightStr + "_" + styleStr;
                         fontDict.put(key, fullFilename);
+                        String name2 = Character.toUpperCase(name.charAt(0)) + name.substring(1);
```

Comment:
I think the uppercase step should happen at style parsing, so that we always ask the operating system with a valid font file name. This would also be more straightforward while implementing this on iOS.